tools/downloader/common.py

```diff
@@ -26,7 +26,7 @@
 DOWNLOAD_TIMEOUT = 5 * 60
 
 # make sure to update the documentation if you modify these
-KNOWN_FRAMEWORKS = {'caffe', 'dldt', 'mxnet', 'tf'}
+KNOWN_FRAMEWORKS = {'caffe', 'dldt', 'mxnet', 'tf', 'pytorch'}
```

Comment:
Keep them in alphabetical order, please. You also need to update the README correspondingly.

Reply:
By "correspondingly" I meant that there's a list of frameworks in the part that talks about Info Dumper's output, which needs to be updated in the same way.